lib/HLSL/DxilPreparePasses.cpp

```diff
@@ -652,7 +652,7 @@ class DxilFinalizeModule : public ModulePass {
           Function *F = CI->getParent()->getParent();
           ICmpInst *Cmp = DxBreakCmpMap.lookup(F);
           if (!Cmp) {
-            Instruction *IP = dxilutil::FirstNonAllocaInsertionPt(F);
+            Instruction *IP = dxilutil::FindAllocaInsertionPt(F);
```

Comment:
This is used to create loads and then a compare of the loaded value for a global so it's fine at the top.